pkg/apis/core/types_shoot.go

```diff
@@ -172,6 +174,15 @@ type ShootCARotation struct {
 	LastCompletionTime *metav1.Time
 }
 
+// ShootKubeconfigRotation contains information about the kubeconfig credential rotation.
+type ShootKubeconfigRotation struct {
+	// LastInitiationTime is the most recent time when the certificate authority credential rotation was initiated.
+	LastInitiationTime *metav1.Time
```

Comment:
Nit: Why did you go for "last _initiation_ time"? "Last start time" would probably fit better to the `rotate-ca-start` operation annotation.

Reply:
I didn't like `LastStartedTime` here because it doesn't quite fit well to `LastCompletionTime` (asymmetric) compared to `LastInitiationTime`. We can change `rotate-ca-start` to `rotate-ca-initiate` if you want (separately, not this PR).

Reply:
`LastStartTime` and `LastCompletionTime` would be like `LastUpdateTime` and `LastTransitionTime` in conditions 😄 
Anyways, I'm not strongly voting for it.

Reply:
I think the comparison is not correct - in this case "transition" is the verb. In our case, "complete" would be the verb but we called it "completION" :)

Reply:
Whatever 😄 Let's keep `initiation` then 😄